theano/gof/cmodule.py

```diff
@@ -2000,6 +2025,20 @@ def join_options(init_part):
                     _logger.info("g++ -march=native equivalent flags: %s",
                                  GCC_compiler.march_flags)
 
+            # Find working march flag:
+            default_detected_flag = GCC_compiler.march_flags[0][7:]
```

Comment:
Why the indexing [0][7:]?

I would just use "GCC_compiler.march_flags"

You don't know where march and mtune will be in that set of flags. This won't be consistent on all platform. So you need to find where they are and change them. Also changing just them could still cause the problem users had due to the other flags. So we need to test with just change of march and mtune or just the extra flags. (Or just test the extra flag)